### DISCUSSION TITLE
Gjs(MJML) can't see content formatted using Ckeditor

### DISCUSSION THREAD

**Discussion by VanoAkofashvili** [GrapesJS/grapesjs#5624] [Q&A]
Hi there,
I'm using the `grapesjs-mjml` plugin to build email templates. The user can add custom variables in `mj-text` by focusing it and selecting a variable from the trait, which will take the variable and append it as a child to the component like this:
```js
 onEvent({ component, event }) {
        const selectedTag = (event.target as HTMLInputElement).value
        component.append(`{{${ selectedTag }}}`)
 }
```
I ran into a problem when I changed the rich text editor from default to `Ckeditor`.
When the user focuses on text and selects a custom variable, Canvas is showing `{{variable}}` and existing content, but in fact it's in reversed (correct) order: existing content and `{{variable}}`. See the screenshots:
Before:
<img width="565" alt="image" src="https://github.com/GrapesJS/grapesjs/assets/49734341/a9a52525-a3ef-438f-aceb-dfc1381f0639">
Selecting the variable:
<img width="817" alt="image" src="https://github.com/GrapesJS/grapesjs/assets/49734341/b1b939f0-581d-4f9e-ba40-e838b00f9434">
Result:
<img width="571" alt="image" src="https://github.com/GrapesJS/grapesjs/assets/49734341/01aa1193-9c7c-4f5c-8d7e-5f2c99b31e38">
 But model saves the correct content:

<img width="657" alt="image" src="https://github.com/GrapesJS/grapesjs/assets/49734341/16b11268-b8f3-4eca-9cb3-9943219b3fea">

versions:
grapesjs: 0.21.4
grapesjs-mjml: 1.0.5
grapesjs-plugin-ckeditor: 1.0.1

Config for Ckeditor:
```js
 {
          onToolbar(el) {
            el.style.minWidth = '350px'
          },
          options: {
            colorButton_enableMore: true,
            colorButton_colors: 'ffd106,fff,000',
            startupFocus: true,
            extraAllowedContent: '*(*);*{*}', // Allows any class and any inline style
            allowedContent: true, // Disable auto-formatting, class removing, etc.
            enterMode: 2, // CKEDITOR.ENTER_BR,
            extraPlugins:
              'sharedspace,justify,colorbutton,panelbutton,font,colordialog',
            toolbar: [
              { name: 'styles', items: ['Font', 'FontSize'] },
              ['Bold', 'Italic', 'Underline', 'Strike'],
              { name: 'paragraph', items: ['NumberedList', 'BulletedList'] },
              { name: 'links', items: ['Link', 'Unlink'] },
              { name: 'colors', items: ['TextColor', 'BGColor'] },
              ['JustifyLeft', 'JustifyCenter', 'JustifyRight'],
            ],
          },
        }
```

**Comment by VanoAkofashvili** [comment]
Hi, @artf 
Can you help me on this issue, please?

**Comment by AngieSweeft** [comment]
@artf I have a same issue, please advice

**Comment by AngieSweeft** [comment]
@artf Help please
